### ISSUE TITLE
Fix invalid read in rtcpNackListGet

### ISSUE THREAD

**Issue by Sean-Der** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#156]
```
==19521== Process terminating with default action of signal 11 (SIGSEGV)                                                                                                                                                                   
==19521==  Bad permissions for mapped region at address 0x26CE6000                                                                                                                                                                         
==19521==    at 0x486AC37: rtcpNackListGet (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                                                
==19521==    by 0x4865ECD: resendPacketOnNack (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                                             
==19521==    by 0x48663A7: onRtcpPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                                                   
==19521==    by 0x48637C6: onInboundPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                                                
==19521==    by 0x4858631: incomingDataHandler (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                                            
==19521==    by 0x4854F2B: connectionListenerReceiveDataRoutine (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)                                                                                           
==19521==    by 0x5119FB6: start_thread (pthread_create.c:486)                                                                                                                                                                             
==19521==    by 0x4C912CE: clone (clone.S:95)  


==21619== 
==21619== Process terminating with default action of signal 11 (SIGSEGV)
==21619==  Access not within mapped region at address 0x16232000
==21619==    at 0x486AC37: rtcpNackListGet (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x4865ECD: resendPacketOnNack (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x48663A7: onRtcpPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x48637C6: onInboundPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x4858631: incomingDataHandler (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x4854F2B: connectionListenerReceiveDataRoutine (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==21619==    by 0x5119FB6: start_thread (pthread_create.c:486)
==21619==    by 0x4C912CE: clone (clone.S:95)


2020-01-23 22:20:51 DEBUG   turnConnectionCheckTurnBeingUsed(): Detected turn connection not being used, start freeing turn allocation
2020-01-23 22:20:51 DEBUG   turnConnectionStepState(): TurnConnection state changed from TURN_STATE_READY to TURN_STATE_CLEAN_UP
2020-01-23 22:20:51 DEBUG   turnConnectionStepState(): TurnConnection state changed from TURN_STATE_CREATE_PERMISSION to TURN_STATE_BIND_CHANNEL
==24067== 
==24067== Process terminating with default action of signal 11 (SIGSEGV)
==24067==  Bad permissions for mapped region at address 0x26E34000
==24067==    at 0x486AC37: rtcpNackListGet (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x4865ECD: resendPacketOnNack (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x48663A7: onRtcpPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x48637C6: onInboundPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x4858631: incomingDataHandler (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x4854F2B: connectionListenerReceiveDataRoutine (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==24067==    by 0x5119FB6: start_thread (pthread_create.c:486)
==24067==    by 0x4C912CE: clone (clone.S:95)


==25092== Conditional jump or move depends on uninitialised value(s)
==25092==    at 0x486ACC5: rtcpNackListGet (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4865ECD: resendPacketOnNack (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x48663A7: onRtcpPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x48637C6: onInboundPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4858631: incomingDataHandler (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4854F2B: connectionListenerReceiveDataRoutine (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x5119FB6: start_thread (pthread_create.c:486)
==25092==    by 0x4C912CE: clone (clone.S:95)
==25092== 
==25092== 
==25092== Process terminating with default action of signal 11 (SIGSEGV)
==25092==  Access not within mapped region at address 0x19C7D000
==25092==    at 0x486AC37: rtcpNackListGet (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4865ECD: resendPacketOnNack (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x48663A7: onRtcpPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x48637C6: onInboundPacket (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4858631: incomingDataHandler (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x4854F2B: connectionListenerReceiveDataRoutine (in /root/amazon-kinesis-video-streams-webrtc-sdk-c/build/libkvsWebrtcClient.so)
==25092==    by 0x5119FB6: start_thread (pthread_create.c:486)
==25092==    by 0x4C912CE: clone (clone.S:95)
```

**Comment by Sean-Der** [comment]
@zhiyua-git If you have the bandwidth mind doing the memset? That should fix this for now!

We can grab the rsize thing soon, but for sure falls behind other things like congestion control (IMO)

**Comment by Sean-Der** [comment]
Fixed by #164
